src/main/java/io/pinecone/clients/AsyncIndex.java

```diff
@@ -192,6 +192,13 @@ public ListenableFuture<DeleteResponse> delete(List<String> ids,
         return asyncStub.delete(deleteRequest);
     }
 
+    @Override
+    public ListenableFuture<DescribeIndexStatsResponse> describeIndexStats() {
```

Comment:
Just a convenience for allowing users to call it without passing an explicit `null`.